include/ruby/protect.hpp

```diff
@@ -2,14 +2,20 @@
 
 #include <ruby/type.hpp>
 
+#if __cplusplus <= 199711L
+# define RB_EXTPP_NORETURN
+#else
+# define RB_EXTPP_NORETURN [[noreturn]]
+#endif
+
```

Comment:
We can always use `[[noreturn]]` because we can assume C++11 or later.

Reply:
OK, I'll fix it.

Reply:
Fixed.